<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiLayer</class>
 <widget class="QWidget" name="UiLayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>138</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>136</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>138</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="layer_name">
       <property name="toolTip">
        <string>Название слоя</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>21</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>21</width>
         <height>20</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Идентификатор слоя</string>
       </property>
       <property name="text">
        <string>Ид: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lid">
       <property name="minimumSize">
        <size>
         <width>31</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>31</width>
         <height>16</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Идентификатор слоя</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Тип преобразования:&lt;/p&gt;&lt;p&gt;  *  Не изменять - брать готовые изображения как есть&lt;/p&gt;&lt;p&gt;  *  Замена цвета - перебрать цвет сгенерированные случайно или по заданной палитре. Требует настройки&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Изменение:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mode_choose">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Тип преобразования:&lt;/p&gt;&lt;p&gt;  *  Не изменять - брать готовые изображения как есть&lt;/p&gt;&lt;p&gt;  *  Замена цвета - перебрать цвет сгенерированные случайно или по заданной палитре. Требует настройки&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <property name="placeholderText">
        <string>...</string>
       </property>
       <property name="modelColumn">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Не изменять</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Замена цвета</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="settings">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Настройка цветовой замены</string>
       </property>
       <property name="text">
        <string>⚙</string>
       </property>
       <property name="icon">
        <iconset resource="../res.qrc">
         <normaloff>:/image/icon/resourses/gear.ico</normaloff>:/image/icon/resourses/gear.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="path_label">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string>Файлы:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="path_edit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="inputMask">
        <string notr="true"/>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="path_buton">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Редактировать в диалоге</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../res.qrc">
         <normaloff>:/image/icon/resourses/edit.ico</normaloff>:/image/icon/resourses/edit.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="probe_label">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>20</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Вероятность использования слоя в каждом выходном изображении.
От 0 до 1</string>
       </property>
       <property name="text">
        <string>Шанс использовать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="probe_val">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Вероятность использования слоя в каждом выходном изображении.
От 0 до 1</string>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>0.001000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="blinked">
       <property name="minimumSize">
        <size>
         <width>97</width>
         <height>17</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Есвли включено, то при генерации будет, как-будто, добавлена пустая картинка.
</string>
       </property>
       <property name="text">
        <string>Пропадающий</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>1</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="noise_label">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Вероятность использовать 1 раз  слой с принудительной заменой цвета на белый шум.
Значение от 0 до 1</string>
       </property>
       <property name="text">
        <string>Шанс шума</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="noise_probe">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Вероятность использовать 1 раз  слой с принудительной заменой цвета на белый шум.
Значение от 0 до 1</string>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
